frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureStrategyMenu.tsx

```diff
@@ -33,17 +33,16 @@ interface IFeatureStrategyMenuProps {
     matchWidth?: boolean;
     size?: IPermissionButtonProps['size'];
     disableReason?: string;
+    allowReleasePlanFeedback?: boolean;
```

Comment:
If it's passed in one place, maybe it should be slot-like instead of boolean? That's very optional.

Reply:
I missed this 'cos I set this to auto merge (wanted this to go out asap if it was good) but that's a really good idea! Totally do that tomorrow!